scripts/process_schemas.py

```diff
@@ -311,19 +313,44 @@ def convertAvro(self, avdlFile):
             with open(os.devnull, 'w') as devnull:
                 subprocess.check_call(args, stdout=devnull, stderr=devnull)
 
-    def run(self):
-        url = "https://github.com/ga4gh/schemas/archive/{0}.tar.gz".format(
-            self.version)
-        self.download(url, self.sourceTar)
-        with tarfile.open(self.sourceTar, "r") as tarball:
-            tarball.extractall(self.tmpDir)
-        directory = os.path.join(self.schemaDir, "src/main/resources/avro")
+    def setupAvroJar(self):
         if self.avroJarPath is not None:
             self.avroJar = os.path.abspath(self.avroJarPath)
         else:
             url = "http://www.carfab.com/apachesoftware/avro/stable/java/"\
                 "avro-tools-1.7.7.jar"
             self.download(url, self.avroJar)
+
+    def schemaFromGitHub(self):
+        """
+        Downloads a tagged version of the schemas
+        from the official GitHub repo.
+        """
+        url = "https://github.com/ga4gh/schemas/archive/{0}.tar.gz".format(
+            self.version)
+        self.download(url, self.sourceTar)
+        with tarfile.open(self.sourceTar, "r") as tarball:
+            tarball.extractall(self.tmpDir)
+
+    def schemaFromLocal(self):
+        """
+        Copies schemas from a local directory.
+        """
+        destDir = os.path.join(self.schemaDir, self.avroPath)
+        if not os.path.exists(destDir):
+            os.makedirs(destDir)
+        avdlFiles = glob.iglob(os.path.join(self.sourceDir, "*.avdl"))
+        for avdlFile in avdlFiles:
+            if os.path.isfile(avdlFile):
+                shutil.copy2(avdlFile, destDir)
+
+    def run(self):
+        if self.sourceDir is None:
+            self.schemaFromGitHub()
+        else:
+            self.schemaFromLocal()
```

Comment:
It would be a little easier to understand if these were called `getSchemaFromGitHub` and `getSchemaFromLocal`.